src/components/Queue.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+class Queue extends React.Component {
+  render() {
+    return (
+      <ul className="queue">
+        <li>1</li>
+        <li>2</li>
+        <li>3</li>
+        <li>4</li>
+        <li>5</li>
+      </ul>
```

Comment:
I'm not sure if I'll have two lists in each queue, one for the grid of blocks in the background and one for the monsters? Not sure how animating would work.